recipes/screen_capture_lite/all/patches/0001-fix-cmake.patch

```diff
@@ -0,0 +1,168 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d646b23..fe28583 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,4 +1,4 @@
+-cmake_minimum_required(VERSION 3.20)
++cmake_minimum_required(VERSION 3.12)
+ project(screen_capture_lite_build)
+ 
+ set(CMAKE_CXX_STANDARD 20)
+@@ -22,14 +22,21 @@ else()
+ endif()
+ 
+ add_subdirectory(src_cpp) 
+-add_subdirectory(src_csharp) 
+ 
+-install (TARGETS screen_capture_lite_static screen_capture_lite_shared 
+-	RUNTIME DESTINATION bin
+-	ARCHIVE DESTINATION lib
+-	LIBRARY DESTINATION lib
+-)
+- 
++if(BUILD_SHARED_LIBS)
++  install (TARGETS screen_capture_lite_shared
++    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
++    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
++  )
++else()
++  install (TARGETS screen_capture_lite_static
++    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
++    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
++  )
++endif()
++
+ install (FILES 
+ 	include/ScreenCapture.h 
+ 	DESTINATION include
+diff --git a/src_cpp/CMakeLists.txt b/src_cpp/CMakeLists.txt
+index bc59c0b..2b173d5 100644
+--- a/src_cpp/CMakeLists.txt
++++ b/src_cpp/CMakeLists.txt
+@@ -1,28 +1,28 @@
+ project(screen_capture_lite)
+ 
+ if(WIN32)
+-    set(SCREEN_CAPTURE_PLATFORM_SRC     
++    set(SCREEN_CAPTURE_PLATFORM_SRC
```

Comment:
Are white space changes really required 🦖 🤔

Reply:
Oh. It's my mistake. I am going to revert it.

Reply:
Fixed